docs/installation/server-config/security.rst

```diff
@@ -187,50 +196,51 @@ Standard ACLs
     - ``DESCRIBE`` permission on the ``__consumer_offsets`` topic.
 
 Input topics
-    The authenticated KSQL user requires ``DESCRIBE`` and ``READ`` permissions for each input topic. The topic should already exist
-    when KSQL is started.
+    An input topic is a topic that is imported into KSQL using a ``CREATE STREAM`` or ``CREATE TABLE``
```

Comment:
suggest: `An input topic is imported into...`

Reply:
done